src/components/FindInvite.js

```diff
@@ -0,0 +1,168 @@
+// import {findUser} from "../firebase/db";
+import {Form, Button, Col, InputGroup, FormControl} from "react-bootstrap";
+import React, {useState} from "react";
+import {db} from "../firebase/firebase";
+
+const FindInvite = () => {
+  const [guest, setGuest] = useState({firstName: "", lastName: ""});
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [existingGuest, setExistingGuest] = useState({});
+  const [rsvp, setRsvp] = useState("");
+  const [plusOne, setPlusOne] = useState(false);
+  const [plusFName, setPlusFName] = useState("");
+  const [plusLName, setPlusLName] = useState("");
+  const [allergies, setAllergies] = useState("");
+  const [guestEmail, setEmail] = useState("");
+
+  const clearFields = () => {
+    setFirstName("");
+    setLastName("");
+  };
+
+  const handleSubmit = (e) => {
+    const guestData = {
+      firstName: firstName,
+      lastName: lastName,
+    };
+
+    e.preventDefault();
+    setGuest(guestData);
+    findUser(guest);
+    clearFields();
+  };
+
+  const findUser = async (guest) => {
+    try {
+      const guests = db.collection("guests").doc(`${guest.firstName} ${guest.lastName}`);
+      const doc = await guests.get();
+      if (doc) {
+        console.log(doc.data());
+        setExistingGuest(doc.data());
+      } else {
+        console.log("no data");
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const handlePlusOne = (e) => {
+    const status = e.target.checked;
+    console.log(e);
+    setPlusOne(status);
+  };
+
+  return (
+    <Form onSubmit={(e) => handleSubmit(e)}>
```

Comment:
Might be a good idea to pass the `e.target.checked` as the parameter to stay consistent with the others. This may remove the need for the `handlePlusOne` function entirely